c2rust-transpile/src/translator/mod.rs

```diff
@@ -488,7 +484,7 @@ fn clean_path(mod_names: &RefCell<IndexMap<String, PathBuf>>, path: Option<&path
                 .parent()
                 .unwrap()
                 .iter()
-                .map(|os| PathBuf::from(os))
+                .map(PathBuf::from)
```

Comment:
An `|s| s.into()` would work, too.  Generally `.into()` is preferred over `::from()`, but I'm not sure when you can avoid the closure.

Reply:
I prefer using `From` where it doesn't result in convoluted code. It makes the type clear, and it is easy to use as a free function. But eventually it depends on the context.

Reply:
I think it's fine here.